js/views/components/moderators/Moderators.js

```diff
@@ -403,19 +403,16 @@ export default class extends baseVw {
   render() {
     const state = this.getState();
     const showMods = this.modCards.filter(mod => this.modShouldRender(mod.model));
-    const unVerCount = this.modCards.filter(mod =>
-      mod.model.hasModCurrency(state.showOnlyCur) && !mod.model.isVerified).length;
-    const totalIDs = this.allIDs.length;
+    const unVerCount = this.modCards.filter(mod => (!state.showOnlyCur ||
+      mod.model.hasModCurrency(state.showOnlyCur)) && !mod.model.isVerified).length;
```

Comment:
Since a default currency isn't being passed in (removed below), this was updated to be count all the unverified moderators if there is no currency set.